Sources/ArcGISToolkit/Documentation.docc/Tutorials/JobManagerTutorial.tutorial

```diff
@@ -0,0 +1,138 @@
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep1) {
+                    @Image(source: JobManagerTakeMapOfflineButton, alt: "An Image of an iPhone app with a Take Map Offline button.")
+                }
+            }
+            
+            @Step {
+                Add two extensions containing helper functions for making the job.
+                
+                The first function gets the map from the portal and calls the second function.  The second function uses a task to make the generate offline map job.
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep2, previousFile: JobManagerViewStep1)
+            }
+            
+            @Step {
+                Make the UI that will show when a job has been added to the job manager.
+                
+                To begin,add a state variable for the job status. Just after the check to see if we have a job, put a progress view that will show the progress of the job. Next, add a button remove the job from the job manager and start over; give it the title "Start New Job". Last add a task to keep track of the job status in the status state variable.
+                
+                @Code(name: "JobManagerTutorialView.swift", file: JobManagerViewStep3, previousFile: JobManagerViewStep2) {
+                    @Image(source: JobManagerTutorial, alt: "An Image of an iPhone app with a progress bar and Start New Job button.")
+                }
+            }
+            
+            @Step {
+                Let's look at what we have so far. When the "Take Map Offline" button is tapped, a job is added to the job manager and the job is started.
+                
+                The first thing the job does is connect to the specified server to kick off the process that will make a mobile map package. Jobs are intended to be used with longer running service tasks. Once the task on the server is started the job waits and polls.
+
+                @Image(source: JobManagerTutorialJobStart, alt: "An image of the app UI with a progress bar four percent done.")
+            }
+            
+            @Step {
+                To know when the work on the server is done, the job polls the service. The job periodically requests the status of the work the service is doing.
+                
+                For this tutorial, the service is collecting all the parts of a map and putting them together to create a mobile map package.
+                
+                @Image(source: JobManagerTutorialJobPoll, alt: "An image of the app UI with a progress bar twenty-one percent done.")
```

Comment:
Very nice transition here from the last image!

Reply:
Thanks